tests/subresource_api_test.go

```diff
@@ -135,7 +135,7 @@ var _ = Describe("[sig-compute]Subresource Api", func() {
 			})
 
 			It("[test_id:1305][posneg:negative] should return an error when VM is not running", func() {
-				vm := tests.NewRandomVirtualMachine(tests.NewRandomVMI(), false)
+				vm := tests.NewRandomVirtualMachine(libvmi.New(), false)
 				vm, err := virtCli.VirtualMachine(util.NamespaceTestDefault).Create(vm)
 				Expect(err).NotTo(HaveOccurred())
```

Comment:
is it the same error as before, or a new one because you did not specify requested memory?

Reply:
Hey Dan,
It's the same error as before.